docusaurus/docs/create-a-plugin/extend-a-plugin/add-authentication-for-data-source-plugins.md

```diff
@@ -0,0 +1,486 @@
+---
+id: add-authentication-for-data-source-plugins
+title: Add authentication for data source plugins
+sidebar_position: 2
+description: How to add authentication for data source plugins.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - authentication
+  - data source
+  - datasource
+---
+
+# Add authentication for data source plugins
+
+Grafana plugins can perform authenticated requests against a third-party API by using the _data source proxy_ or through a custom a _backend plugin_.
```

Comment:
curious whether italics are in keeping with the writers guide? not a blocker or anything

Reply:
yes, italic for emphasis is okay, bold isn't. not in Writers' Toolkit, but incorporated by reference to Google at https://developers.google.com/style/text-formatting.